src/exchange/disclaimer/disclaimer.html

```diff
@@ -19,13 +19,13 @@
                   data-uib-tooltip="{{'exchange_tab_DISCLAIMER_disclaimer_outside_only_tooltip' | translate}}"></span>
         </oui-column>
         <oui-column data-title="'exchange_tab_DISCLAIMER_state_header' | translate">
-            <span class="label label-success"
+            <span class="oui-status oui-status_info"
```

Comment:
```suggestion
            <span class="oui-status oui-status_success"
```

Below you've changed :
`success` to `success`
`warning` to `warning`

Reply:
Hello, it's voluntary, in this case the status must be informative  in place of success